build.rs

```diff
@@ -424,8 +424,8 @@ fn main() {
     }
 
     // To filter away some flaky test (see src/float/add.rs for details)
-    if llvm_target.last() == Some(&"gnueabihf") {
-        println!("cargo:rustc-cfg=gnueabihf")
+    if llvm_target.last().unwrap().contains("gnueabi") {
+        println!("cargo:rustc-cfg=gnueabi")
```

Comment:
Could you use two cfgs here, gnueabi and gnueabihf, instead of overloading the word "gnueabi" to mean both ABIs? I think it would be less error prone that way.